webapp/cypress/support/component-index.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <title>Components App</title>
+    <script src="https://cdn.bokeh.org/bokeh/release/bokeh-2.3.3.min.js"></script>
```

Comment:
```suggestion
    <script src="https://cdn.bokeh.org/bokeh/release/bokeh-2.4.3.min.js"></script>
```

I think we're now on 2.4.3, we may also need https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.4.3.min.js in future